### PR TITLE
Focusable and tabs

### PR DESCRIPTION
**Pull request by julienCarret**

### CODE REVIEW

**Review by jlp0328 — APPROVED** [review]

File: projects/demo/src/app/demo/pages/focusable-page/focusable-page.component.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<pa-demo-page>
+    <pa-demo-title>Focusable directive</pa-demo-title>
+
+    <pa-demo-description>
+        <p><code>paFocusable</code> directive mark any html element as focusable.<br>
+        Browser's default accessibility style is removed and a pastanaga <code>pa-keyboard-focus</code> class is applied to the element when the focus is triggered by a keyboard action.
+        </p><p>
+        <code>paFocusDisabled option allow to toggle tabindex from 0 to -1</code>
+    </p>
```
Comment (mpellerin42) [inline projects/demo/src/app/demo/pages/focusable-page/focusable-page.component.html:9]: indentation

File: projects/pastanaga-angular/src/lib/tabs/tabs-list.component.ts [hunk 2]
```diff
@@ -7,32 +7,43 @@ import {
...
 @Component({
     selector: 'pa-tabs',
     templateUrl: 'tabs-list.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class TabsListComponent implements AfterContentInit, OnDestroy {
+    @Input() set fullWidth(full: boolean) {
+        this.fullWidthTabItems = coerceBooleanProperty(full);
+        this.updateSlider();
+    }
```
Comment (mpellerin42) [inline projects/pastanaga-angular/src/lib/tabs/tabs-list.component.ts:28]: the default is true: full width by default. so the Input should be something allowing to not have full width, so `noFullWidth` for example. I guess in most cases this won't be set dynamically: we want either full width or no full width on an element. Doing it this way will allow us to have code like
```html
<pa-tabs noFullWidth>…
```

File: projects/pastanaga-angular/src/lib/table/table-cell-menu/table-cell-menu.component.spec.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { TableCellMenuComponent } from './table-cell-menu.component';
+import { MockModule } from 'ng-mocks';
+import { PaFocusableModule } from '../../..';
```
Comment (mpellerin42) [inline projects/pastanaga-angular/src/lib/table/table-cell-menu/table-cell-menu.component.spec.ts:5]: wrong import

File: projects/pastanaga-angular/src/lib/table/table.spec.ts [hunk 4]
```diff
@@ -6,6 +6,8 @@ import { TableRowHeaderComponent } from './table-row-header/table-row-header.com
 import { TableCellComponent } from './table-cell/table-cell.component';
 import { TableRowComponent } from './table-row/table-row.component';
 import { TableComponent } from './table.component';
+import { MockModule } from 'ng-mocks';
+import { PaFocusableModule } from '../..';
```
Comment (mpellerin42) [inline projects/pastanaga-angular/src/lib/table/table.spec.ts:10]: wrong import

**Review by mpellerin42 — CHANGES_REQUESTED** [review]

**Review by mpellerin42 — APPROVED** [review]

**Pull request merged**